lago-ovirt.spec.in

```diff
@@ -42,7 +42,6 @@ Requires: python-lago >= 0.39
 Requires: python-magic
 Requires: python-nose
 Requires: ovirt-engine-sdk-python >= 3.6.9.1-1
-Requires: createrepo
 Requires: repoman >= 2.0.12
 Requires: yum-utils
```

Comment:
This is what prevents you from upgrading to F29.

Reply:
Yes, but I need reposync... Or make code changes to use dnf reposync on Fedora and reposync on EL?

Reply:
`dnf-utils` provides `/usr/bin/reposync`.